### ISSUE TITLE
The ECP for future ECPs

### ISSUE THREAD

**Issue by hashratez** [Ether1Project/ECPs#2]
The development of the Ether1 Project has fallen heavily on DevJames, Fallen, Treehouse, Primate and others in the past two years (sorry if I forgot to mention your name).  Going forward in the spirit of Decentralization the project needs to migrate to a more bottom-up approach.  The Community must propose ideas, direction, and improvements.  

Therefore if a Community Member wishes to submit an ECP they must do some research on the improvement.  Which means submitting a budget for the item (if there is a cost associated with the improvement).  For example, a new "Android Wallet" that looks like "XYZ Coin" because that wallet just rocks.  Ok, well maybe XYZ wallet was developed using the ABC Development Kit, and that cost $500.  In addition, the cost of developers needs to be also included because no one at Ether1 knows how to use the ABC kit.  In this case the Community Member got a bid for the developer's time and that is also $500.  

In summary, the Community Member would submit an ECP that says something like this:
```
I, BigKillerDude, propose a new Ether1 Andriod Wallet, I have found the tool kit and developers, the kit is $500 and the developers have given me a bid for $500 and it will take 2 weeks.  Total is $1000.

We need this wallet because.....
```
Submitting an ECP with a budget brings a reality check to the process and also provides discipline to the process.
